src/util/earthEngine.js

```diff
@@ -28,6 +28,18 @@ export const getPeriodFromFilter = (filter = []) => {
         : null;
 };
 
+// Returns period name from filter
+export const getPeriodNameFromFilter = filter => {
+    const period = getPeriodFromFilter(filter);
+
+    if (!period) {
+        return null;
+    }
+
+    const { name, year = '' } = period;
+    return `${name} ${year}`;
```

Comment:
Perhaps do a whitespace trim (i.e., when year is empty string)

Reply:
I'll fix

Reply:
Changed to:

```
const { name, year } = period;
return `${name}${year ? ` ${year}` : ''}`;
```